docs/installation/index.md

```diff
@@ -15,30 +15,59 @@ Linux](https://learn.microsoft.com/en-us/windows/wsl/)
 ## Opam
 
 We need [opam](https://opam.ocaml.org/), the OCaml Package Manager. There are
-many ways to install it depending on your platform, but let's go with the
-simplest method:
+many ways to [install](https://opam.ocaml.org/doc/Install.html) it depending on
+your platform, but let's go with the simplest method:
 
-```bash
+```shell
 bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
 opam init
-eval $(opam env)
 ```
 
 While `opam init` is running, it will prompt you with something like
 
 ```
-Do you want opam to modify ~/.profile?
+Do you want opam to modify ~/.zshrc? [N/y/f]
```

Comment:
why did this change? Users might not have zsh.

Reply:
The previous line says "it will prompt you with something like", I was hoping that would cover it. Do you think it should it be changed to this?

```
Do you want opam to modify ~/<startup file e.g. .zshrc>? [N/y/f]
```

Reply:
No, I think it's ok, I was just curious.